R/mergingandstackingutilities.R

```diff
@@ -106,11 +106,8 @@ writeDataSet <- function(data.set, data.set.name, is.saved.to.cloud)
                            ".zip on the Cloud Drive as it is too large. ",
                            "It needs to be downloaded, unzipped and re-uploaded to be ",
                            "used in a Displayr document.")
-        error.msg <- paste0("Some data files could not be parsed due to invalid characters ",
-                            "in some of the variable names. ",
-                            "These data files can be fixed by inserting them in a Displayr ",
-                            "document amd  exporting them as an SPSS file (.sav) to the ",
-                            " Cloud Drive via the Publish button.")
+        error.msg <- paste0("The data file could not be saved due to invalid characters ",
+                            "in some of the variable names. Please contact support for assistance.")
```

Comment:
We would want to be notified of this if it happens as it means we aren't properly sanitizing the variable names.

Reply:
So the error message gets passed to createExceptionHandler? What does that do? Should we also pass it to StopForUserError so it triggers a bugdup instead of needing the user to contact support?

Reply:
The purpose of `InterceptExceptions` and `createExceptionHandler` is so that when the error "must have valid SPSS variable names" is thrown by the R package used to save the file, we rethrow it as "The data file could not be saved due to invalid characters in some of the variable names. Please contact support for assistance.". We don't want to use `StopForUserError` as that will mean it won't go to bugdup.